### PR TITLE
[JDK21+] Increase wait time in VirtualThreadTests

### PR DESCRIPTION
**Pull request by babsingh**
The wait time has been increased from 500 ms to 10000 ms. This should
give the virtual thread sufficient time to park on slower machines.

This impacts the following tests:
- test_synchronizedBlockFromVirtualthread
- test_jniFromVirtualthread

test_YieldedVirtualThreadGetStackTrace was earlier updated
in #17194.

Fixes: #17924

Signed-off-by: Babneet Singh <sbabneet@ca.ibm.com>

### CODE REVIEW

**Comment by babsingh** [comment]
@pshipton @fengxue-IS Requesting your review.

**Review by fengxue-IS — APPROVED** [review]
lgtm, I am going to assume the Java19andUp version of this test is obsolete.

**Comment by babsingh** [comment]
> I am going to assume the Java19andUp version of this test is obsolete.

Yes, Java19andUp and Java20andUp versions are obsolete since we no longer support the associated Java versions.

File: test/functional/Java21andUp/src/org/openj9/test/jep425/VirtualThreadTests.java [hunk 1]
```diff
@@ -51,6 +51,16 @@ public class VirtualThreadTests {
 
 	public static native boolean lockSupportPark();
 
+	private void incrementalWait(Thread t) throws InterruptedException {
+		/* Incrementally wait for 10000 ms. */
+		for (int i = 0; i < 20; i++) {
```
Comment (pshipton) [inline test/functional/Java21andUp/src/org/openj9/test/jep425/VirtualThreadTests.java:56]: I like the idea. On a reasonable machine is a 500ms wait usually necessary? I'm thinking the wait time could be decreased and the number of iterations increased so the test could normally take less time to complete.

Comment (babsingh) [reply test/functional/Java21andUp/src/org/openj9/test/jep425/VirtualThreadTests.java]: Yes, smaller increments will be better. Reduced the wait time by a factor of 10 and accordingly increased the number of iterations.

**Comment by pshipton** [comment]
Tested in grinder https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/2667/, which passed.

**Review by pshipton — APPROVED** [review]

**Pull request merged**